qgreenland/config/helpers/layers/lonlat.py

```diff
@@ -45,13 +44,14 @@ def _make_lonlat_layer(
             asset=asset,
         ),
         steps=[
-            *ogr2ogr(
-                input_file='{input_dir}/' + f'{asset.filepath.eval().name}',
-                output_file='{output_dir}/' + f'{asset.filepath.eval().stem}.gpkg',
-                boundary_filepath=project.boundaries['background'].filepath,
-                ogr2ogr_args=[
-                    '-segmentize',
-                    f'{segment_max_distance}',
+            # Clip and reproject the dataset
+            CommandStep(
+                args=[
+                    'ogr2ogr',
+                    *STANDARD_OGR2OGR_ARGS,
+                    'where "wgs84Decimal >= 40"',
```

Comment:
Should be `-where`. Does this work anyway?

Reply:
@hsimonoko were you able to test this?

Reply:
I did, it doesn't work. weird that that change didn't make it on here since I know I fixed that. I'll fix again!